Server/controllers/authController.js

```
@@ -14,298 +14,159 @@ import jwt from "jsonwebtoken";
-		// Compare password
-		const match = await user.comparePassword(password);
-		if (match !== true) {
-			next(new Error(errorMessages.AUTH_INCORRECT_PASSWORD));
-			return;
+		if (!(await user.comparePassword(password))) {
+			throw new Error(errorMessages.AUTH_INCORRECT_PASSWORD);
 		}
 
-		// Remove password from user object.  Should this be abstracted to DB layer?
 		const userWithoutPassword = { ...user._doc };
 		delete userWithoutPassword.password;
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)